index.html

```diff
@@ -2978,13 +2977,13 @@ <h2>Compression method 0</h2>
 <p>Only PNG compression method 0 is defined by this International
 Standard. Other values of compression method are reserved for
 future standardization. PNG compression method 0 is
-deflate/inflate compression with a sliding window
+<a>deflate</a> compression with a sliding window
```

Comment:
This was previously "deflate/inflate".
I _think_ when adding the link, you accidentally removed "/inflate".
I question myself there because it happens elsewhere too so maybe it was intentional?

But I believe it should still include the "inflate" wording in all those cases. I'll point them out.